### PR TITLE
Finding largest number

### PR DESCRIPTION
**Pull request by madhupriyaallam**
Implementation to find largest number:
example - To find to score in a test

### CODE REVIEW

File: src/week2/binarySearch/BinarySearch.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package week2.binarySearch;
+
+public class BinarySearch {
+    int n;
+    static final int notAPerfectSquare = -1;
```
Comment (Hemant-GH) [inline src/week2/binarySearch/BinarySearch.java:5]: wrong naming convention

File: src/week2/findLargestValue/Class.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
+package week2.findLargestValue;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class Class {
```
Comment (Hemant-GH) [inline src/week2/findLargestValue/Class.java]: wrong naming convention

File: src/week2/findLargestValue/Test1.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
+package week2.findLargestValue;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Test1 implements HighestScore {
```
Comment (Hemant-GH) [inline src/week2/findLargestValue/Test1.java]: please choose a meaningful name for the class that has some resemblance with interface

**Review by Hemant-GH — CHANGES_REQUESTED** [review]

**Review by Hemant-GH — APPROVED** [review]

File: src/week2/binarySearch/BinarySearch.java (hunk 1 above)
Comment (madhupriyaallam) [reply src/week2/binarySearch/BinarySearch.java:5]: hemant, can you please tell me whats wrong here

**Review by Hemant-GH — APPROVED** [review]

**Pull request merged**